svn/common.py

```diff
@@ -135,6 +135,47 @@ def info(self, rel_path=None, revision=None):
         info['commit_revision'] = info['commit#revision']
 
         return info
+		
+    def get_property(self, property, rel_path=None, revision=None, recursive=False):
+        """ Return an svn property, or, if called recursively, a list of tuples associated with a
+                    relative path.
+                :param property: the property to query
+                :param rel_path: relative path in the svn repo to query the
+                                 property from
+                :param revision: the revision to query the property from
+                :param recursive: flag for recursivity
+                :returns: if recursive, a list of pairs (svn path, property value)
+                otherwise, a single str corresponding to the property value
+		"""
+        args = []
+        if recursive == True:
+            args += ['-R']
+        if revision:
```

Comment:
nit: I'd prefer that we specifically check for it being not None (any value other than the default).

Reply:
I pushed a fix for this in order to speed things up.